ci/build-unix.yml

```diff
@@ -40,3 +44,20 @@ steps:
     env:
       # to connect to bintray
       JFROG_CONFIG_CONTENT: $(JFROG_CONFIG_CONTENT)
+  - bash: |
+      set -euxo pipefail
+      git tag v$(release_tag)
+      git push origin v$(release_tag)
+      mv bazel-genfiles/release/sdk-release-tarball.tar.gz ./daml-sdk-$(release_tag)-${{ parameters.name }}.tar.gz
+    condition: eq(variables['has_released'], 'true')
+  - task: GitHubRelease@0
+    condition: eq(variables['has_released'], 'true')
+    inputs:
+      gitHubConnection: 'garyverhaegen-da'
```

Comment:
What exactly are the implications on the user name here?

Reply:
I know it looks like one, but this is not a username, this is the name of an Azure "connection service" to github. It has my name because I created it; it's not entirely clear to me how much it depends on my account and what happens the day my account disappears.

I could create another connection service in Azure with a more opaque name. However, despite having spent significant time looking into it, I have not found any way to create an Azure connection that is not linked to my account, so I figured we may as well have that explicitly in the name.